Kubernetes/windows/Dockerfile

```diff
@@ -28,5 +28,7 @@ ENV AGENT_VERSION "winakslogprivatepreview"
 ENV OS_TYPE "windows"
 ENV APPLICATIONINSIGHTS_AUTH "OTQzNWI0M2YtOTdkNS00ZGVkLThkOTAtYjA0Nzk1OGU2ZTg3"
 ENV AZMON_COLLECT_ENV False
+ENV CI_CERT_LOCATION "C://oms.crt"
```

Comment:
does that work unix style file paths?

Reply:
yeah, this works. The current running image has it set in this way.